.travis.yml

```diff
@@ -3,8 +3,7 @@ conditions: v1
 
 language: python
 python:
-    - 3.5
-    - 3.6
+    - 3.7
```

Comment:
Please revert this. We need to run tests across a variety of Python versions.

Reply:
Have reverted the removal of the tests on 3.5 & 3.6

Reply:
Now, remove this line, because it's already added under `jobs.include`.

Reply:
Ah I misunderstood... done!